dist/leaflet.css

```diff
@@ -330,6 +330,25 @@ svg.leaflet-image-layer.leaflet-interactive path {
 	border-bottom-right-radius: 2px;
 	}
 
+
+/* Dark Mode */

```

Comment:
@Falke-Design Should we move all dark mode styles to the bottom of this file (but before `@media print` styles) for clarity?